docs/applications/integrate-application-deploy-to-source-provider-ci.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Integrate Application Deploy to Source Provider's CI
+intro: Commit statuses provide a way to tag commits with meta data, like automated deploy results. Learn how to create outgoing webhooks to update a commit status to integrate the application deploy to the source provider's CI.
```

Comment:
This is a post about Devopness outgoing hooks, not about vendor specific jargon `commit status`.
It's ok to give a vendor specific example, such as `GitHub commit status` when creating an actual outgoing hook. However, title and intro should be aligned with the goal of the topic, otherwise readers can easily get confused.